### PR TITLE
[TASK] Add typo3-rector to scoped version

### PR DESCRIPTION
**Pull request by sabbelasichon**

### CODE REVIEW

**Comment by sabbelasichon** [comment]
I have released version 0.11.0 with PHP 8 only. This is really insane Tomas. I know that you are aware of but it should be more visible. I am sitting here in frontend of the computer and could cry and scream for joy.

**Comment by TomasVotruba** [comment]
So happy to hear this :partying_face:  It's a team effort of many skilled and modest people, including you :+1: 

---

<br>

Let's test the deploy in here. Enable this workflows - https://github.com/rectorphp/rector-src/blob/8ce853d8ff7a264909f59b7cf7909ed82e0bf310/.github/workflows/build_scoped_rector.yaml#L5 - for a pull_request. Then we should see this branch being deployed to http://github.com/rectorphp/rector

**Comment by sabbelasichon** [comment]
Don´t understand why it is failing.

**Comment by TomasVotruba** [comment]
Neither do I :D we'll figure it out.

The new package must be added here:
https://github.com/rectorphp/rector-src/blob/8ce853d8ff7a264909f59b7cf7909ed82e0bf310/utils/compiler/src/Command/DowngradePathsCommand.php#L34

File: utils/compiler/src/Command/DowngradePathsCommand.php [hunk 1]
```diff
@@ -41,6 +41,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             'vendor/symfony vendor/psr',
             'vendor/symplify vendor/nikic bin src packages rector.php',
             'rules',
+            'vendor/helmich/typo3-typoscript-parser/src',
+            'vendor/symfony/var-exporter'
```
Comment (TomasVotruba) [inline utils/compiler/src/Command/DowngradePathsCommand.php:45]: The https://packagist.org/packages/ssch/typo3-rector should be added too

Comment (TomasVotruba) [reply utils/compiler/src/Command/DowngradePathsCommand.php:45]: Oh, it seems added in `findVendorAndRulePaths()` already, so good :+1:  sorry

**Comment by sabbelasichon** [comment]
Should i try to remove the composer require ssch/typo3-rector and see if it works. Maybe it is a side effect.

**Comment by TomasVotruba** [comment]
It will be somethign related to scoper... I'll try it locally

**Comment by sabbelasichon** [comment]
Closing this in favour of #46

**Pull request closed without merging**